src/views/pages/dashboard/invoice/DashboardInvoicePageTs.ts

```diff
@@ -29,33 +30,9 @@ import FormTransferTransaction from '@/views/forms/FormTransferTransaction/FormT
 // @ts-ignore
 import failureIcon from '@/views/resources/img/monitor/failure.png'
 
-// @TODO: to move out
-/**
- * Mosaic object to be displayed in the views
- * @export
- * @interface BalanceEntry
- */
-export interface BalanceEntry {
-  /**
-   * Mosaic Id
-   * @type {MosaicId}
-   */
-  id: MosaicId
-  /**
-   * Mosaic hex Id
-   * @type {string}
-   */
-  mosaicHex: string
-  /**
-   * Mosaic name
-   * @type {string}
-   */
-  name: string
-  /**
-   * Relative amount
-   * @type {number}
-   */
-  amount: number
+export interface ITransaction {
+  transaction: TransferTransaction
+  mosaicAttachment: MosaicAttachmentType[]
```

Comment:
rename field to `attachments` (and should be plural)

Reply:
ok,thanks